src/i18n/fr_FR.ts

```diff
@@ -616,6 +616,84 @@ Si plusieurs demandes échouent dans un court laps de temps, cela peut être dû
             special:
                 'Remorqueuse nécessaire (FPT, FPTL, CCFS, CCFM, CCFL, CCRL, CCRM, CCGC, FMOGP, VLHR, CDHR, FDGP)',
         },
+        39: {
+            caption: 'VR',
+            coins: 10,
+            credits: 10_000,
+            minPersonnel: 0,
+            maxPersonnel: 6,
+            possibleBuildings: [11],
+        },
+        40: {
+            caption: 'VCT',
+            coins: 10,
+            credits: 25_000,
+            minPersonnel: 0,
+            maxPersonnel: 4,
+            possibleBuildings: [11],
+            schooling: {
+                'Poste de police': {
+                    ['Commandement Mobile']: {
+                        all: true,
+                    },
+                },
+            },
+        },
+        41: {
+            caption: 'RAM',
+            coins: 10,
+            credits: 25_000,
+            minPersonnel: 0,
+            maxPersonnel: 0,
```

Comment:
Max is 2
![image](https://user-images.githubusercontent.com/1109899/156630878-ea04c298-8a5e-49ff-ad5e-debc483a983d.png)

Reply:
thank you 
I have modified